Makefile

```diff
@@ -115,7 +75,31 @@ help:
 .pre-fleetctl:
 	$(eval APP_NAME = fleetctl)
 
-build: fleet fleetctl
+BINS_TO_BUILD = fleet fleetctl
+ifdef FLEET
+	BINS_TO_BUILD = fleet
+else ifdef FLEETCTL
+	BINS_TO_BUILD = fleetctl
+endif
```

Comment:
Allow building either fleet or fleetctl or both, all through the `build` command (so you can do `fdm build --fleet` or `fdm build --fleetctl`, or just `fdm build` to do both)